src/CosmosCache.cs

```diff
@@ -215,6 +258,7 @@ private static CosmosCacheSession BuildCosmosCacheSession(string key, byte[] con
                 SessionKey = key,
                 Content = content,
                 TimeToLive = timeToLive,
+                IsSlidingExpiration = timeToLive.HasValue ? options.SlidingExpiration.HasValue : (bool?)null,
```

Comment:
Why not:
```suggestion
                IsSlidingExpiration = timeToLive.HasValue && options.SlidingExpiration.HasValue,
```

Reply:
No real reason other then to have `IsSlidingExpiration` set to null if `TimeToLive` was null so they would sort of be in sync with each other. But logically it isn't required to be null. If we use your suggestion then we could also change `IsSlidingExpiration` to be `bool` instead of `bool?`. Would you also prefer that too?

Reply:
Honestly the only thing that threw me off was the `(bool?)null`, it made understanding a tad bit difficult. I don't mind the `bool?` type.

Reply:
Just to clarify. If the value is being set on some code path, like this one, I would either set it as `true` or `false`, setting it to `(bool?)null` is kind of an implicit `false` which is harder to understand. I'd leave the `null` possibility for a scenario where the value does not exist in the source like I mentioned in my previous comment (code migration to a version that contains the attribute).

Reply:
ok sounds good